invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/transition.overrides

```diff
@@ -2,6 +2,26 @@
     Invenio App RDM Transition Overrides
 ***********************************************/
 
+#carousel-slides {
```

Comment:
there is no carousel anymore in invenio-app-rdm so maybe is better to move these in zenodo-rdm? Or are these changes needed for the existing macros?

Reply:
All the styling is for the macros and/or react component in `invenio-communities`. `zenodo-rdm` is just importing the macro from `invenio-communities`